templates/activity/parts/activity-effects.hbs

```diff
@@ -3,18 +3,43 @@
     <multi-select name="appliedEffects">
         {{ selectOptions allEffects }}
     </multi-select>
-    <ul class="unlist">
-        {{#each activity.effects}}
-        <li data-index="{{ @index }}" data-effect-id="{{ id }}" class="flexrow">
-            <img class="gold-icon" src="{{ effect.img }}" alt="{{ effect.name }}">
-            <span class="name">{{{ dnd5e-linkForUuid effect.uuid }}}</span>
-            <button type="button" class="unbutton" data-action="deleteEffect">
-                {{ localize "DND5E.EFFECT.Action.Delete" }}
-            </button>
+    <ul class="separated-list effects-list">
+        {{#each appliedEffects}}
+        <li data-index="{{ @index }}" data-effect-id="{{ id }}" data-expand-id="effects.{{ id }}">
+            {{> ".effect" }}
         </li>
         {{/each}}
     </ul>
     <button type="button" class="unbutton" data-action="addEffect">
         {{ localize "DND5E.EFFECT.Action.Create" }}
     </button>
 </fieldset>
+
+{{#*inline ".effect"}}
+<div class="details flexrow">
+    <img class="gold-icon" src="{{ effect.img }}" alt="{{ effect.name }}">
+    {{{ dnd5e-linkForUuid effect.uuid }}}
+    <div class="list-controls flexrow">
+        <button type="button" class="unbutton" data-action="deleteEffect"
+                data-tooltip="DND5E.EFFECT.Action.Delete"
+                aria-label="{{ localize 'DND5E.EFFECT.Action.Delete' }}">
+            <i class="fa-solid fa-trash fa-fw" inert></i>
+        </button>
+    </div>
+</div>
+<input type="hidden" name="{{ prefix }}id" value="{{ data.id }}">
```

Comment:
Missing a dot, but maybe this should be `_id` as well? Not sure.
```suggestion
<input type="hidden" name="{{ prefix }}.id" value="{{ data.id }}">
```

Reply:
The prefix contains the dot `effects.${index}.`.